### PR TITLE
Admin duplicate

### PR DESCRIPTION
**Pull request by cmangeat**
linked with https://github.com/camptocamp/c2cgeoform/pull/146

### CODE REVIEW

File: admin/acceptance_tests/__init__.py [hunk 1]
```diff
@@ -65,3 +65,9 @@ def check_checkboxes(self, form, name, expected):
             assert exp['label'] == list(label.stripped_strings)[0]
             assert exp['value'] == checkbox['value']
             assert exp['checked'] == field.checked
+
+    def getFirstFieldNamed(self, form, name):
```
Comment (arnaud-morvan) [inline admin/acceptance_tests/__init__.py]: the standard in python is to use underscores in function names

File: admin/acceptance_tests/__init__.py [hunk 2]
```diff
@@ -65,3 +65,9 @@ def check_checkboxes(self, form, name, expected):
...
+        return form.fields.get(name)[0]
+
+    def setFirstFieldNamed(self, form, name, value):
+        form.fields.get(name)[0].value__set(value)
```
Comment (arnaud-morvan) [inline admin/acceptance_tests/__init__.py]: Note that I do not find these functions really useful

Comment (cmangeat) [reply admin/acceptance_tests/__init__.py]: They ought to be stub for further improvement...
For example getFirstFieldNamed(self, form, name) could become checkFirstFieldNamed((self, form, name, expected) .
I had to replace a few "form[name] = value" by "form.fields.get(name)[0] = value" then by "form.fields.get(name)[0].value__set(value)", so I thought I could be a good thing to be able to do the replacement in one place.

File: admin/c2cgeoportal_admin/views/layers_wms.py [hunk 3]
```diff
@@ -50,19 +50,25 @@ def index(self):
...
 
-    @view_config(route_name='c2cgeoform_action',
+    @view_config(route_name='c2cgeoform_item',
                  request_method='DELETE')
     def delete(self):
         return super().delete()
+
+    @view_config(route_name='c2cgeoform_item_action',
+                 request_method='GET',
+                 renderer='../templates/edit.jinja2')
```
Comment (arnaud-morvan) [inline admin/c2cgeoportal_admin/views/layers_wms.py]: There is no filter on duplicate action, maybe we should add a match_param filter here ?

Comment (cmangeat) [reply admin/c2cgeoportal_admin/views/layers_wms.py]: True, it works because at this time there is one and only one action

**Pull request merged**